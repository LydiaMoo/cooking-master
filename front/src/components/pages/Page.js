import React, { Component } from 'react';
import styled from 'styled-components';
import { Header, Nav, Newsfeed, Footer } from '../organisms';

const Main = styled.div`
  background: #FFFFFF; 
  display: flex;
  flex-direction: column; 
  font-family: 'Roboto', sans-serif;
`;

// const names = ['Lydia', 'Loopy', 'Lolly'];
// names[Math.floor(Math.random() * Math.floor(3))]

class Page extends Component {

  state = {
    filter: 'all', 
    search: ''
  }

  handleFiltersOnChange = (filter) => {
    this.setState({filter})
    this.setState({search : ''})
  }

  handleSearchOnChange = (search) => {
    this.setState({search})
    this.setState({filter : 'all'})
  }


  render() {
    return (
      <Main>
        <Header/> 
        <Nav 
          filtersOnChange={this.handleFiltersOnChange}
          searchOnChange={this.handleSearchOnChange}
          dropDownSelect={this.handleSearchOnChange}
        /> 
        <Newsfeed 
          filter={this.state.filter}
          search={this.state.search}
        />
        <Footer
          name='lydia'
        />
      </Main>
    );
  }
}

export default Page; 