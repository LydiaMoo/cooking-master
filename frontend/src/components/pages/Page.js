import React, { Component } from 'react';
import styled from 'styled-components';
import { Header, Nav, Newsfeed, Footer } from '../organisms';

const Main = styled.div`
  background: #FFFFFF; 
  display: flex;
  flex-direction: column; 
  font-family: 'Roboto', sans-serif;
`;

class Page extends Component {

  state = {
    filter: 'all', 
    search: ''
  }

  handleFiltersOnChange = (filter) => {
    this.setState({filter})
    this.setState({search : ''})
  }

  handleSearchOnChange = (search) => {
    this.setState({search})
    this.setState({filter : 'all'})
  }

  render() {
    return (
      <Main>
        <Header/> 
        <Nav 
          filtersOnChange={this.handleFiltersOnChange}
          searchOnChange={this.handleSearchOnChange}
        /> 
        <Newsfeed 
          filter={this.state.filter}
          search={this.state.search}
        />
        <Footer
          name='Lydia'
        />
      </Main>
    );
  }
}

export default Page; 