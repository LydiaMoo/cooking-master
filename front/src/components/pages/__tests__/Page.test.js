import React, { Component } from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Page from '../Page';

describe('<Page />', () => {

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Page />, div);
  });

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Page filter="starred"/>, div);
  });

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Page search=""/>, div);
  });

  it('should match default snapshot', () => {
    const tree = renderer.create(<Page />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should match default snapshot', () => {
    const tree = renderer.create(<Page filter="starred" />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should match default snapshot', () => {
    const tree = renderer.create(<Page filter="all" />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should match default snapshot', () => {
    const tree = renderer.create(<Page search="lemon" />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should match default snapshot', () => {
    const tree = renderer.create(<Page search="" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

