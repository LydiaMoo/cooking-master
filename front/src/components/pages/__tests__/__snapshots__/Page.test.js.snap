// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Page /> should match default snapshot 1`] = `
<div
  className="sc-kpOJdX gYvxyg"
>
  <header
    className="sc-bdVaJa kLCyua"
  >
    <h1>
      Cooking Master
      <div
        className="sc-bwzfXH bBScxE"
      >
        <span
          aria-label="chef"
          role="img"
        >
          üë©üèª‚Äçüç≥
        </span>
      </div>
    </h1>
  </header>
  <nav
    className="sc-htoDjs hjaVFT"
  >
    <ul
      className="sc-htpNat dMshuV"
    >
      <li
        className="sc-bxivhb jbfCJm"
        data-test-id="allBtn"
        onClick={[Function]}
      >
        <span
          className="sc-ifAKCX kYeoyA"
        >
          All
        </span>
      </li>
      <li
        className="sc-bxivhb jbfCJm"
      >
        <img
          alt="starred"
          className="sc-EHOje gAKosP"
          data-test-id="starredBtn"
          onClick={[Function]}
          src="Star.svg"
        />
      </li>
    </ul>
    <select
      className="sc-gzVnrw eoZEAD"
      onChange={[Function]}
    >
      <option
        value="Mushroom"
      >
        Mushroom
      </option>
      <option
        value="Chicken"
      >
        Chicken
      </option>
      <option
        value="Beef"
      >
        Beef
      </option>
    </select>
    <form
      className="sc-bZQynM dKMWjK"
      onSubmit={[Function]}
    >
      <input
        onChange={[Function]}
        placeholder="Ingredient, Recipe Name"
        type="text"
      />
      <input
        type="submit"
        value="Submit"
      />
    </form>
  </nav>
  <div>
    <div>
      <article
        className="sc-kGXeez gmamMX"
      >
        <article
          className="sc-gqjmRU cwbCPD"
        >
          <div
            className="sc-VigVT eaARUA"
          >
            <h1>
              Lemon Chicken
            </h1>
            <button
              className="sc-dnqmqq fOAjwu"
              onClick={[Function]}
            >
              <img
                alt="not a favorite Recipe"
                className="sc-gZMcBi ekOSZp"
                src="EmptyStar.svg"
              />
            </button>
          </div>
          <img
            alt="Lemon Chicken"
            className="sc-jTzLTM eAPKWj"
            src="http://assets.simplyrecipes.com/wp-content/uploads/2010/03/lemon-chicken-horiz-a-1600.jpg"
          />
          <button
            className="sc-fjdhpX gOIRDV"
            onClick={[Function]}
          >
            Click to see more
          </button>
        </article>
        <article
          className="sc-gqjmRU cwbCPD"
        >
          <div
            className="sc-VigVT eaARUA"
          >
            <h1>
              Beef Stroganoff
            </h1>
            <button
              className="sc-dnqmqq fOAjwu"
              onClick={[Function]}
            >
              <img
                alt="not a favorite Recipe"
                className="sc-gZMcBi ekOSZp"
                src="EmptyStar.svg"
              />
            </button>
          </div>
          <img
            alt="Beef Stroganoff"
            className="sc-jTzLTM eAPKWj"
            src="http://www.daringgourmet.com/wp-content/uploads/2014/12/Slow-Cooker-Beef-Stroganoff-2-web-edited.jpg"
          />
          <button
            className="sc-fjdhpX gOIRDV"
            onClick={[Function]}
          >
            Click to see more
          </button>
        </article>
        <article
          className="sc-gqjmRU cwbCPD"
        >
          <div
            className="sc-VigVT eaARUA"
          >
            <h1>
              Ceasar Salad
            </h1>
            <button
              className="sc-dnqmqq fOAjwu"
              onClick={[Function]}
            >
              <img
                alt="not a favorite Recipe"
                className="sc-gZMcBi ekOSZp"
                src="EmptyStar.svg"
              />
            </button>
          </div>
          <img
            alt="Ceasar Salad"
            className="sc-jTzLTM eAPKWj"
            src="https://www.bbcgoodfood.com/sites/default/files/styles/recipe/public/recipe_images/recipe-image-legacy-id--327831_11.jpg?itok=8GjuBS4F"
          />
          <button
            className="sc-fjdhpX gOIRDV"
            onClick={[Function]}
          >
            Click to see more
          </button>
        </article>
        <article
          className="sc-gqjmRU cwbCPD"
        >
          <div
            className="sc-VigVT eaARUA"
          >
            <h1>
              Chicken Ceasar Salad
            </h1>
            <button
              className="sc-dnqmqq fOAjwu"
              onClick={[Function]}
            >
              <img
                alt="not a favorite Recipe"
                className="sc-gZMcBi ekOSZp"
                src="EmptyStar.svg"
              />
            </button>
          </div>
          <img
            alt="Chicken Ceasar Salad"
            className="sc-jTzLTM eAPKWj"
            src="https://www.bbcgoodfood.com/sites/default/files/styles/recipe/public/recipe_images/recipe-image-legacy-id--327831_11.jpg?itok=8GjuBS4F"
          />
          <button
            className="sc-fjdhpX gOIRDV"
            onClick={[Function]}
          >
            Click to see more
          </button>
        </article>
      </article>
    </div>
  </div>
  <footer
    style={
      Object {
        "background": "#635155",
        "color": "white",
        "display": "flex",
        "justifyContent": "center",
        "padding": "2%",
      }
    }
  >
    Made by 
    lydia
  </footer>
</div>
`;

exports[`<Page /> should match default snapshot 2`] = `
<div
  className="sc-kpOJdX gYvxyg"
>
  <header
    className="sc-bdVaJa kLCyua"
  >
    <h1>
      Cooking Master
      <div
        className="sc-bwzfXH bBScxE"
      >
        <span
          aria-label="chef"
          role="img"
        >
          üë©üèª‚Äçüç≥
        </span>
      </div>
    </h1>
  </header>
  <nav
    className="sc-htoDjs hjaVFT"
  >
    <ul
      className="sc-htpNat dMshuV"
    >
      <li
        className="sc-bxivhb jbfCJm"
        data-test-id="allBtn"
        onClick={[Function]}
      >
        <span
          className="sc-ifAKCX kYeoyA"
        >
          All
        </span>
      </li>
      <li
        className="sc-bxivhb jbfCJm"
      >
        <img
          alt="starred"
          className="sc-EHOje gAKosP"
          data-test-id="starredBtn"
          onClick={[Function]}
          src="Star.svg"
        />
      </li>
    </ul>
    <select
      className="sc-gzVnrw eoZEAD"
      onChange={[Function]}
    >
      <option
        value="Mushroom"
      >
        Mushroom
      </option>
      <option
        value="Chicken"
      >
        Chicken
      </option>
      <option
        value="Beef"
      >
        Beef
      </option>
    </select>
    <form
      className="sc-bZQynM dKMWjK"
      onSubmit={[Function]}
    >
      <input
        onChange={[Function]}
        placeholder="Ingredient, Recipe Name"
        type="text"
      />
      <input
        type="submit"
        value="Submit"
      />
    </form>
  </nav>
  <div>
    <div>
      <article
        className="sc-kGXeez gmamMX"
      >
        <article
          className="sc-gqjmRU cwbCPD"
        >
          <div
            className="sc-VigVT eaARUA"
          >
            <h1>
              Lemon Chicken
            </h1>
            <button
              className="sc-dnqmqq fOAjwu"
              onClick={[Function]}
            >
              <img
                alt="not a favorite Recipe"
                className="sc-gZMcBi ekOSZp"
                src="EmptyStar.svg"
              />
            </button>
          </div>
          <img
            alt="Lemon Chicken"
            className="sc-jTzLTM eAPKWj"
            src="http://assets.simplyrecipes.com/wp-content/uploads/2010/03/lemon-chicken-horiz-a-1600.jpg"
          />
          <button
            className="sc-fjdhpX gOIRDV"
            onClick={[Function]}
          >
            Click to see more
          </button>
        </article>
        <article
          className="sc-gqjmRU cwbCPD"
        >
          <div
            className="sc-VigVT eaARUA"
          >
            <h1>
              Beef Stroganoff
            </h1>
            <button
              className="sc-dnqmqq fOAjwu"
              onClick={[Function]}
            >
              <img
                alt="not a favorite Recipe"
                className="sc-gZMcBi ekOSZp"
                src="EmptyStar.svg"
              />
            </button>
          </div>
          <img
            alt="Beef Stroganoff"
            className="sc-jTzLTM eAPKWj"
            src="http://www.daringgourmet.com/wp-content/uploads/2014/12/Slow-Cooker-Beef-Stroganoff-2-web-edited.jpg"
          />
          <button
            className="sc-fjdhpX gOIRDV"
            onClick={[Function]}
          >
            Click to see more
          </button>
        </article>
        <article
          className="sc-gqjmRU cwbCPD"
        >
          <div
            className="sc-VigVT eaARUA"
          >
            <h1>
              Ceasar Salad
            </h1>
            <button
              className="sc-dnqmqq fOAjwu"
              onClick={[Function]}
            >
              <img
                alt="not a favorite Recipe"
                className="sc-gZMcBi ekOSZp"
                src="EmptyStar.svg"
              />
            </button>
          </div>
          <img
            alt="Ceasar Salad"
            className="sc-jTzLTM eAPKWj"
            src="https://www.bbcgoodfood.com/sites/default/files/styles/recipe/public/recipe_images/recipe-image-legacy-id--327831_11.jpg?itok=8GjuBS4F"
          />
          <button
            className="sc-fjdhpX gOIRDV"
            onClick={[Function]}
          >
            Click to see more
          </button>
        </article>
        <article
          className="sc-gqjmRU cwbCPD"
        >
          <div
            className="sc-VigVT eaARUA"
          >
            <h1>
              Chicken Ceasar Salad
            </h1>
            <button
              className="sc-dnqmqq fOAjwu"
              onClick={[Function]}
            >
              <img
                alt="not a favorite Recipe"
                className="sc-gZMcBi ekOSZp"
                src="EmptyStar.svg"
              />
            </button>
          </div>
          <img
            alt="Chicken Ceasar Salad"
            className="sc-jTzLTM eAPKWj"
            src="https://www.bbcgoodfood.com/sites/default/files/styles/recipe/public/recipe_images/recipe-image-legacy-id--327831_11.jpg?itok=8GjuBS4F"
          />
          <button
            className="sc-fjdhpX gOIRDV"
            onClick={[Function]}
          >
            Click to see more
          </button>
        </article>
      </article>
    </div>
  </div>
  <footer
    style={
      Object {
        "background": "#635155",
        "color": "white",
        "display": "flex",
        "justifyContent": "center",
        "padding": "2%",
      }
    }
  >
    Made by 
    lydia
  </footer>
</div>
`;

exports[`<Page /> should match default snapshot 3`] = `
<div
  className="sc-kpOJdX gYvxyg"
>
  <header
    className="sc-bdVaJa kLCyua"
  >
    <h1>
      Cooking Master
      <div
        className="sc-bwzfXH bBScxE"
      >
        <span
          aria-label="chef"
          role="img"
        >
          üë©üèª‚Äçüç≥
        </span>
      </div>
    </h1>
  </header>
  <nav
    className="sc-htoDjs hjaVFT"
  >
    <ul
      className="sc-htpNat dMshuV"
    >
      <li
        className="sc-bxivhb jbfCJm"
        data-test-id="allBtn"
        onClick={[Function]}
      >
        <span
          className="sc-ifAKCX kYeoyA"
        >
          All
        </span>
      </li>
      <li
        className="sc-bxivhb jbfCJm"
      >
        <img
          alt="starred"
          className="sc-EHOje gAKosP"
          data-test-id="starredBtn"
          onClick={[Function]}
          src="Star.svg"
        />
      </li>
    </ul>
    <select
      className="sc-gzVnrw eoZEAD"
      onChange={[Function]}
    >
      <option
        value="Mushroom"
      >
        Mushroom
      </option>
      <option
        value="Chicken"
      >
        Chicken
      </option>
      <option
        value="Beef"
      >
        Beef
      </option>
    </select>
    <form
      className="sc-bZQynM dKMWjK"
      onSubmit={[Function]}
    >
      <input
        onChange={[Function]}
        placeholder="Ingredient, Recipe Name"
        type="text"
      />
      <input
        type="submit"
        value="Submit"
      />
    </form>
  </nav>
  <div>
    <div>
      <article
        className="sc-kGXeez gmamMX"
      >
        <article
          className="sc-gqjmRU cwbCPD"
        >
          <div
            className="sc-VigVT eaARUA"
          >
            <h1>
              Lemon Chicken
            </h1>
            <button
              className="sc-dnqmqq fOAjwu"
              onClick={[Function]}
            >
              <img
                alt="not a favorite Recipe"
                className="sc-gZMcBi ekOSZp"
                src="EmptyStar.svg"
              />
            </button>
          </div>
          <img
            alt="Lemon Chicken"
            className="sc-jTzLTM eAPKWj"
            src="http://assets.simplyrecipes.com/wp-content/uploads/2010/03/lemon-chicken-horiz-a-1600.jpg"
          />
          <button
            className="sc-fjdhpX gOIRDV"
            onClick={[Function]}
          >
            Click to see more
          </button>
        </article>
        <article
          className="sc-gqjmRU cwbCPD"
        >
          <div
            className="sc-VigVT eaARUA"
          >
            <h1>
              Beef Stroganoff
            </h1>
            <button
              className="sc-dnqmqq fOAjwu"
              onClick={[Function]}
            >
              <img
                alt="not a favorite Recipe"
                className="sc-gZMcBi ekOSZp"
                src="EmptyStar.svg"
              />
            </button>
          </div>
          <img
            alt="Beef Stroganoff"
            className="sc-jTzLTM eAPKWj"
            src="http://www.daringgourmet.com/wp-content/uploads/2014/12/Slow-Cooker-Beef-Stroganoff-2-web-edited.jpg"
          />
          <button
            className="sc-fjdhpX gOIRDV"
            onClick={[Function]}
          >
            Click to see more
          </button>
        </article>
        <article
          className="sc-gqjmRU cwbCPD"
        >
          <div
            className="sc-VigVT eaARUA"
          >
            <h1>
              Ceasar Salad
            </h1>
            <button
              className="sc-dnqmqq fOAjwu"
              onClick={[Function]}
            >
              <img
                alt="not a favorite Recipe"
                className="sc-gZMcBi ekOSZp"
                src="EmptyStar.svg"
              />
            </button>
          </div>
          <img
            alt="Ceasar Salad"
            className="sc-jTzLTM eAPKWj"
            src="https://www.bbcgoodfood.com/sites/default/files/styles/recipe/public/recipe_images/recipe-image-legacy-id--327831_11.jpg?itok=8GjuBS4F"
          />
          <button
            className="sc-fjdhpX gOIRDV"
            onClick={[Function]}
          >
            Click to see more
          </button>
        </article>
        <article
          className="sc-gqjmRU cwbCPD"
        >
          <div
            className="sc-VigVT eaARUA"
          >
            <h1>
              Chicken Ceasar Salad
            </h1>
            <button
              className="sc-dnqmqq fOAjwu"
              onClick={[Function]}
            >
              <img
                alt="not a favorite Recipe"
                className="sc-gZMcBi ekOSZp"
                src="EmptyStar.svg"
              />
            </button>
          </div>
          <img
            alt="Chicken Ceasar Salad"
            className="sc-jTzLTM eAPKWj"
            src="https://www.bbcgoodfood.com/sites/default/files/styles/recipe/public/recipe_images/recipe-image-legacy-id--327831_11.jpg?itok=8GjuBS4F"
          />
          <button
            className="sc-fjdhpX gOIRDV"
            onClick={[Function]}
          >
            Click to see more
          </button>
        </article>
      </article>
    </div>
  </div>
  <footer
    style={
      Object {
        "background": "#635155",
        "color": "white",
        "display": "flex",
        "justifyContent": "center",
        "padding": "2%",
      }
    }
  >
    Made by 
    lydia
  </footer>
</div>
`;

exports[`<Page /> should match default snapshot 4`] = `
<div
  className="sc-kpOJdX gYvxyg"
>
  <header
    className="sc-bdVaJa kLCyua"
  >
    <h1>
      Cooking Master
      <div
        className="sc-bwzfXH bBScxE"
      >
        <span
          aria-label="chef"
          role="img"
        >
          üë©üèª‚Äçüç≥
        </span>
      </div>
    </h1>
  </header>
  <nav
    className="sc-htoDjs hjaVFT"
  >
    <ul
      className="sc-htpNat dMshuV"
    >
      <li
        className="sc-bxivhb jbfCJm"
        data-test-id="allBtn"
        onClick={[Function]}
      >
        <span
          className="sc-ifAKCX kYeoyA"
        >
          All
        </span>
      </li>
      <li
        className="sc-bxivhb jbfCJm"
      >
        <img
          alt="starred"
          className="sc-EHOje gAKosP"
          data-test-id="starredBtn"
          onClick={[Function]}
          src="Star.svg"
        />
      </li>
    </ul>
    <select
      className="sc-gzVnrw eoZEAD"
      onChange={[Function]}
    >
      <option
        value="Mushroom"
      >
        Mushroom
      </option>
      <option
        value="Chicken"
      >
        Chicken
      </option>
      <option
        value="Beef"
      >
        Beef
      </option>
    </select>
    <form
      className="sc-bZQynM dKMWjK"
      onSubmit={[Function]}
    >
      <input
        onChange={[Function]}
        placeholder="Ingredient, Recipe Name"
        type="text"
      />
      <input
        type="submit"
        value="Submit"
      />
    </form>
  </nav>
  <div>
    <div>
      <article
        className="sc-kGXeez gmamMX"
      >
        <article
          className="sc-gqjmRU cwbCPD"
        >
          <div
            className="sc-VigVT eaARUA"
          >
            <h1>
              Lemon Chicken
            </h1>
            <button
              className="sc-dnqmqq fOAjwu"
              onClick={[Function]}
            >
              <img
                alt="not a favorite Recipe"
                className="sc-gZMcBi ekOSZp"
                src="EmptyStar.svg"
              />
            </button>
          </div>
          <img
            alt="Lemon Chicken"
            className="sc-jTzLTM eAPKWj"
            src="http://assets.simplyrecipes.com/wp-content/uploads/2010/03/lemon-chicken-horiz-a-1600.jpg"
          />
          <button
            className="sc-fjdhpX gOIRDV"
            onClick={[Function]}
          >
            Click to see more
          </button>
        </article>
        <article
          className="sc-gqjmRU cwbCPD"
        >
          <div
            className="sc-VigVT eaARUA"
          >
            <h1>
              Beef Stroganoff
            </h1>
            <button
              className="sc-dnqmqq fOAjwu"
              onClick={[Function]}
            >
              <img
                alt="not a favorite Recipe"
                className="sc-gZMcBi ekOSZp"
                src="EmptyStar.svg"
              />
            </button>
          </div>
          <img
            alt="Beef Stroganoff"
            className="sc-jTzLTM eAPKWj"
            src="http://www.daringgourmet.com/wp-content/uploads/2014/12/Slow-Cooker-Beef-Stroganoff-2-web-edited.jpg"
          />
          <button
            className="sc-fjdhpX gOIRDV"
            onClick={[Function]}
          >
            Click to see more
          </button>
        </article>
        <article
          className="sc-gqjmRU cwbCPD"
        >
          <div
            className="sc-VigVT eaARUA"
          >
            <h1>
              Ceasar Salad
            </h1>
            <button
              className="sc-dnqmqq fOAjwu"
              onClick={[Function]}
            >
              <img
                alt="not a favorite Recipe"
                className="sc-gZMcBi ekOSZp"
                src="EmptyStar.svg"
              />
            </button>
          </div>
          <img
            alt="Ceasar Salad"
            className="sc-jTzLTM eAPKWj"
            src="https://www.bbcgoodfood.com/sites/default/files/styles/recipe/public/recipe_images/recipe-image-legacy-id--327831_11.jpg?itok=8GjuBS4F"
          />
          <button
            className="sc-fjdhpX gOIRDV"
            onClick={[Function]}
          >
            Click to see more
          </button>
        </article>
        <article
          className="sc-gqjmRU cwbCPD"
        >
          <div
            className="sc-VigVT eaARUA"
          >
            <h1>
              Chicken Ceasar Salad
            </h1>
            <button
              className="sc-dnqmqq fOAjwu"
              onClick={[Function]}
            >
              <img
                alt="not a favorite Recipe"
                className="sc-gZMcBi ekOSZp"
                src="EmptyStar.svg"
              />
            </button>
          </div>
          <img
            alt="Chicken Ceasar Salad"
            className="sc-jTzLTM eAPKWj"
            src="https://www.bbcgoodfood.com/sites/default/files/styles/recipe/public/recipe_images/recipe-image-legacy-id--327831_11.jpg?itok=8GjuBS4F"
          />
          <button
            className="sc-fjdhpX gOIRDV"
            onClick={[Function]}
          >
            Click to see more
          </button>
        </article>
      </article>
    </div>
  </div>
  <footer
    style={
      Object {
        "background": "#635155",
        "color": "white",
        "display": "flex",
        "justifyContent": "center",
        "padding": "2%",
      }
    }
  >
    Made by 
    lydia
  </footer>
</div>
`;

exports[`<Page /> should match default snapshot 5`] = `
<div
  className="sc-kpOJdX gYvxyg"
>
  <header
    className="sc-bdVaJa kLCyua"
  >
    <h1>
      Cooking Master
      <div
        className="sc-bwzfXH bBScxE"
      >
        <span
          aria-label="chef"
          role="img"
        >
          üë©üèª‚Äçüç≥
        </span>
      </div>
    </h1>
  </header>
  <nav
    className="sc-htoDjs hjaVFT"
  >
    <ul
      className="sc-htpNat dMshuV"
    >
      <li
        className="sc-bxivhb jbfCJm"
        data-test-id="allBtn"
        onClick={[Function]}
      >
        <span
          className="sc-ifAKCX kYeoyA"
        >
          All
        </span>
      </li>
      <li
        className="sc-bxivhb jbfCJm"
      >
        <img
          alt="starred"
          className="sc-EHOje gAKosP"
          data-test-id="starredBtn"
          onClick={[Function]}
          src="Star.svg"
        />
      </li>
    </ul>
    <select
      className="sc-gzVnrw eoZEAD"
      onChange={[Function]}
    >
      <option
        value="Mushroom"
      >
        Mushroom
      </option>
      <option
        value="Chicken"
      >
        Chicken
      </option>
      <option
        value="Beef"
      >
        Beef
      </option>
    </select>
    <form
      className="sc-bZQynM dKMWjK"
      onSubmit={[Function]}
    >
      <input
        onChange={[Function]}
        placeholder="Ingredient, Recipe Name"
        type="text"
      />
      <input
        type="submit"
        value="Submit"
      />
    </form>
  </nav>
  <div>
    <div>
      <article
        className="sc-kGXeez gmamMX"
      >
        <article
          className="sc-gqjmRU cwbCPD"
        >
          <div
            className="sc-VigVT eaARUA"
          >
            <h1>
              Lemon Chicken
            </h1>
            <button
              className="sc-dnqmqq fOAjwu"
              onClick={[Function]}
            >
              <img
                alt="not a favorite Recipe"
                className="sc-gZMcBi ekOSZp"
                src="EmptyStar.svg"
              />
            </button>
          </div>
          <img
            alt="Lemon Chicken"
            className="sc-jTzLTM eAPKWj"
            src="http://assets.simplyrecipes.com/wp-content/uploads/2010/03/lemon-chicken-horiz-a-1600.jpg"
          />
          <button
            className="sc-fjdhpX gOIRDV"
            onClick={[Function]}
          >
            Click to see more
          </button>
        </article>
        <article
          className="sc-gqjmRU cwbCPD"
        >
          <div
            className="sc-VigVT eaARUA"
          >
            <h1>
              Beef Stroganoff
            </h1>
            <button
              className="sc-dnqmqq fOAjwu"
              onClick={[Function]}
            >
              <img
                alt="not a favorite Recipe"
                className="sc-gZMcBi ekOSZp"
                src="EmptyStar.svg"
              />
            </button>
          </div>
          <img
            alt="Beef Stroganoff"
            className="sc-jTzLTM eAPKWj"
            src="http://www.daringgourmet.com/wp-content/uploads/2014/12/Slow-Cooker-Beef-Stroganoff-2-web-edited.jpg"
          />
          <button
            className="sc-fjdhpX gOIRDV"
            onClick={[Function]}
          >
            Click to see more
          </button>
        </article>
        <article
          className="sc-gqjmRU cwbCPD"
        >
          <div
            className="sc-VigVT eaARUA"
          >
            <h1>
              Ceasar Salad
            </h1>
            <button
              className="sc-dnqmqq fOAjwu"
              onClick={[Function]}
            >
              <img
                alt="not a favorite Recipe"
                className="sc-gZMcBi ekOSZp"
                src="EmptyStar.svg"
              />
            </button>
          </div>
          <img
            alt="Ceasar Salad"
            className="sc-jTzLTM eAPKWj"
            src="https://www.bbcgoodfood.com/sites/default/files/styles/recipe/public/recipe_images/recipe-image-legacy-id--327831_11.jpg?itok=8GjuBS4F"
          />
          <button
            className="sc-fjdhpX gOIRDV"
            onClick={[Function]}
          >
            Click to see more
          </button>
        </article>
        <article
          className="sc-gqjmRU cwbCPD"
        >
          <div
            className="sc-VigVT eaARUA"
          >
            <h1>
              Chicken Ceasar Salad
            </h1>
            <button
              className="sc-dnqmqq fOAjwu"
              onClick={[Function]}
            >
              <img
                alt="not a favorite Recipe"
                className="sc-gZMcBi ekOSZp"
                src="EmptyStar.svg"
              />
            </button>
          </div>
          <img
            alt="Chicken Ceasar Salad"
            className="sc-jTzLTM eAPKWj"
            src="https://www.bbcgoodfood.com/sites/default/files/styles/recipe/public/recipe_images/recipe-image-legacy-id--327831_11.jpg?itok=8GjuBS4F"
          />
          <button
            className="sc-fjdhpX gOIRDV"
            onClick={[Function]}
          >
            Click to see more
          </button>
        </article>
      </article>
    </div>
  </div>
  <footer
    style={
      Object {
        "background": "#635155",
        "color": "white",
        "display": "flex",
        "justifyContent": "center",
        "padding": "2%",
      }
    }
  >
    Made by 
    lydia
  </footer>
</div>
`;
